### DISCUSSION TITLE
Auto select whole bullets when selection covers more than one bullet

### DISCUSSION THREAD

**Discussion by dcoales** [vslinko/obsidian-outliner#105] [Ideas]
To make cutting a pasting bullets a little easier would it be possible to mimic other outliners, like Dynalist and WorkFlowy, so that if I start a selection in the middle of a bullet, and drag the selection up or down onto another bullet, then the selection would automatically adjust to the start of the first bullet to the end of the last bullet (including all children).

Excellent plugin btw.  Made a huge difference to the usability of Obsidian and is one of the changes that made me come back after I had tried it before going back to WorkFlowy because the outlining wasn't good enough.

**Comment by vslinko** [comment]
@dcoales Thank you for your idea!

I'll consider this in #3 but no promises :)
